modules/ROOT/pages/editor-plugin-version.adoc

```diff
@@ -1,160 +1,121 @@
-= Specify editor & plugin versions
-:description_short: Specifying editor and plugin versions for Tiny Cloud deployments.
-:description: Specifying editor and plugin versions for Tiny Cloud deployments.
+= Specify editor version & plugins
+:description_short: Specifying editor version and plugins for {cloudname} deployments.
+:description: Instructions on setting the {productname} editor version and plugins for {cloudname} deployments.
 :keywords: tinymce, cloud, script, textarea, apiKey, hybrid
 
 [[specifying-the-tinymce-editor-version-deployed-from-cloud]]
-== Specifying the TinyMCE editor version deployed from Cloud
+== Specifying the {productname} Editor Version from the Cloud
 
-Use the URL provided to specify the {productname} version when deploying via {cloudname}. Refer to the xref:editor-and-features.adoc[{productname} editor via the {cloudname}] for more information.
-
-The following example is the default for loading {productname} {productmajorversion} via {cloudname}. Substitute 'no-api-key' with your api key in the examples below.
+To set the {productname} version when deploying from {cloudname}, use the following URL:
 
 [source,html,subs="attributes+"]
 ----
 <script src="{cdnurl}" referrerpolicy="origin"></script>
 ----
 
-This URL specifies the latest and quality assured release of {productname}.
+The example above shows the default way to load {productname} {productmajorversion} from {cloudname}. This URL ensures the most recent and verified version of {productname} is used.
 
-=== Selecting specific version numbers
+[TIP]
+Replace 'no-api-key' with your own API key.
 
-NOTE: All {cloudname} channels are based on the {enterpriseversion} version. For information on the latest version of the {cloudname} `{productmajorversion}` release channel, see: xref:release-notes.adoc[{productname} Release Notes]. For a list of changes that *may* be present in the {cloudname} testing channel, see: xref:changelog.adoc[{productname} Changelog].
+For more details, check out the xref:editor-and-features.adoc[{productname} editor via {cloudname}].
 
-[#{productmajorversion}-{productmajorversion}-testing-and-{productmajorversion}-dev-release-channels]
-=== {productmajorversion}, {productmajorversion}-testing, and {productmajorversion}-dev release channels
+[[specifying-the-cloud-deployment-of-plugins]]
+== Specifying the Cloud Deployment of Plugins
 
-Choose from the `{productmajorversion}`, `{productmajorversion}-testing`, or `{productmajorversion}-dev` release channels to load the latest version of {productname} from {cloudname}.
+When deploying {productname}, integrators have the choice to **self-host** premium plugins or load them from {companyname} Cloud (a.k.a. hybrid mode). Depending on your requirements, you can use a combination of both approaches.
```

Comment:
Hopefully this will avoid mistakes like attempting to load both the editor and plugins from the cloud (which isn't necessary because the editor includes the plugins)
```suggestion
When self-hosting {productname}, integrators have the choice to host premium plugins or load them from {companyname} Cloud (a.k.a. hybrid mode). Depending on your requirements, you can use a combination of both approaches.
```

Reply:
Just to confirm, do we refer to it as hybrid, or do we use any other term.

Reply:
it's hybrid.